rootfs_linux.go

```diff
@@ -62,6 +69,21 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd string) error {
+	if cmd != "" {
+		args := strings.Split(cmd, " ")
+		mCmd := exec.Command(args[0], args[1:]...)
```

Comment:
`return exec.Command(args[0], args[1:]...).CombinedOutput()` it makes sense to check output of command on error.